src/Look/TrailLook.cpp

```diff
@@ -107,7 +123,7 @@ TrailLook::Initialise(const TrailSettings &settings)
     else
       iwidth = std::max(minwidth,
                         (i - NUMSNAILCOLORS / 2) *
-                        Layout::ScalePenWidth(16u) / NUMSNAILCOLORS);
+                        Layout::ScalePenWidth(8u) / NUMSNAILCOLORS);
```

Comment:
Is this change intended? This would affect all trails, no?

Reply:
It would. That's not acceptable.

Reply:
Well, it was intended for my personal use, I forgot to remove it from the commit.
I'll take care of that ASAP

Reply:
ok, reverted the file to its original status.